lightning_client.go

```diff
@@ -520,6 +528,25 @@ const (
 	FullyResolvedChannelUpdate
 )
 
+type OpenStatusUpdate struct {
```

Comment:
@guggero I did not include an `enum` on the `OpenStatusUpdate` struct as a non-nil field indicates the type of the update.

Reply:
SGTM :+1: 
Two nits here: The struct itself needs a Godoc comment. And the comments for the fields aren't valid Godoc comments as they need to start with the field name.